apps/meeting/src/containers/MeetingProviderWrapper/index.tsx

```diff
@@ -108,8 +110,8 @@ const MeetingProviderWrapper: React.FC = () => {
     }
     console.log(`Using ${filterCPUUtilization} background blur and replacement`);
     return (
-      <BackgroundBlurProvider options={{ filterCPUUtilization }}>
-        <BackgroundReplacementProvider options={{ imageBlob, filterCPUUtilization }}>
+      <BackgroundBlurProvider options={{ filterCPUUtilization, logger }}>
+        <BackgroundReplacementProvider options={{ imageBlob, filterCPUUtilization, logger }}>
```

Comment:
https://github.com/aws/amazon-chime-sdk-component-library-react/blob/a2d7461a8a55aafdce9b6b7704e7d79c9451522e/src/providers/BackgroundBlurProvider/index.tsx#L54

it uses the useLogger internally, are we sure this is the issue?

Reply:
think it's because here https://github.com/aws/amazon-chime-sdk-component-library-react/blob/a2d7461a8a55aafdce9b6b7704e7d79c9451522e/src/providers/BackgroundBlurProvider/index.tsx#L162